Create lang-resources page

minimalsm opened the pull request:
## Description
Creates a page for non-English community suggested resources

## Related Issue
None

## Preview
https://ethereumorgwebsitedev01-addlangresourcespage.gtsb.io/en/community/language-resources

lukassim commented on src/content/community/language-resources/index.md:
```diff
@@ -0,0 +1,106 @@
+- [~~Investing.com](http://br.investing.com/news/cryptocurrency-news/) - Brazilian version of Investing.com, regularly covering cryptocurrency news~~
+- [~~InfoMoney](http://www.infomoney.com.br/tudo-sobre/criptomoedas/) - portal for inverstors with a section on cryptocurrency news and articles~~
+- [~~Money Times](http://www.moneytimes.com.br/cryptotimes/) - cryptocurrency news and market tracker~~
+- [~~Exame](http://www.exame.com/noticias-sobre/criptomoedas/) - investor education and news, with a ‘Cryptocurrency news’ subpage~~
```
@minimalsm I guess we should remove the above 4 lines, right?

minimalsm replied:
Done!

minimalsm commented on src/content/community/language-resources/index.md:
```diff
@@ -0,0 +1,106 @@
+- [BeInCrypto](http://www.beincrypto.com.br) - cryptocurrency news and articles, including a list of exchanges, available in Brazil
+- [Cointelegraph](http://cointelegraph.com.br/category/analysis) - Brazilian version of Cointelegraph, a major cryptocurrency news outlet
+- [Seudinheiro](http://www.seudinheiro.com/criptomoedas/) - cryptocurrency news and reports
+- [~~Investing.com](http://br.investing.com/news/cryptocurrency-news/) - Brazilian version of Investing.com, regularly covering cryptocurrency news~~
+- [~~InfoMoney](http://www.infomoney.com.br/tudo-sobre/criptomoedas/) - portal for inverstors with a section on cryptocurrency news and articles~~
+- [~~Money Times](http://www.moneytimes.com.br/cryptotimes/) - cryptocurrency news and market tracker~~
+- [~~Exame](http://www.exame.com/noticias-sobre/criptomoedas/) - investor education and news, with a ‘Cryptocurrency news’ subpage~~
```
```suggestion
```

minimalsm commented:
Todo after we approve this:

[ ] - Add breadcrumbs
[ ] - Add link out to those (in community nav dropdown?)

corwintines reviewed (commented):
This is great, were we going to link out to this anywhere on the website, or just have this be a page we share?

samajammin commented on src/content/community/language-resources/index.md:
```diff
@@ -0,0 +1,102 @@
+
+Know of an educational resource in your language? [Open an issue](https://github.com/ethereum/ethereum-org-website/issues/new/choose) to add it to the list!
+
+If you are bilingual and want to help us reach more people, you can also get involved with the [ethereum.org Translation Program](https://ethereum.org/en/contributing/translation-program/#translation-program) and help us translate the website.
```
```suggestion
If you are bilingual and want to help us reach more people, you can also get involved with the [ethereum.org Translation Program](/contributing/translation-program/#translation-program) and help us translate the website.
```

samajammin commented on src/content/community/language-resources/index.md:
```diff
@@ -0,0 +1,102 @@
+lang: en
+---
+
+# Language resources {#language-resources}
```
Great intro section! I think we should make sure we call out the fact we do translate ethereum.org & link to https://ethereum.org/en/languages/ in case people somehow land on this page & aren't aware of this.

samajammin commented on src/content/community/language-resources/index.md:
```diff
@@ -0,0 +1,102 @@
+
+Hundreds of thousands of Ethereum enthusiasts gather in these online forums to share news, talk about recent developments, debate technical issues, and imagine the future.
+
+## Community resources {#community}
```
Related to my comment above, perhaps we add a H2 section about this on ethereum.org resources / languages, where we mention that we have a bunch of on-site translations.

wackerow replied:
@samajammin Clarifying, are you suggesting adding a section to the `/translations` page, calling this page out?

samajammin replied:
@wackerow I think that's a good idea too! So I'd request 2 changes.

1. Add an H2 on this page - "Ethereum.org resources" that explains our translation program & that we translate all our content into many langues, with a link to our /languages page.
2. Add a section on the /languages page the mentions & links to this page (e.g. "In addition to translating ethereum.org content, we also maintain a curated list of language resources...").

wackerow commented on src/content/community/language-resources/index.md:
```diff
@@ -0,0 +1,102 @@
+
+Our aim is to provide educational content in all languages and help overcome the language barriers that make onboarding people from all over the world to Ethereum a challenge.
+
+If you prefer reading in your native language or know someone who doesn’t speak English, you can find a list of useful non-English resources below.
```
```suggestion
If you prefer reading in your native language or know someone who doesn’t speak English, you can find a list of useful non-English resources below, or you can check if [ethereum.org is available in your native language](/languages).
```

wackerow replied:
or maybe:
```suggestion
If you prefer reading in your native language or know someone who doesn’t speak English, you can find a list of useful non-English resources below. Ethereum.org is also natively translated into dozens of languages which are listed on our [languages page](/languages).
```

wackerow reviewed (commented):
@minimalsm This looks great... Left a couple suggestions on adding a comment for users linking to the `/languages` page, and had a clarifying question for Sam, otherwise this looks ready to go =)

minimalsm commented:
Think this is good to go: 

- Added paragraph to /languages/ page linking to this new page
- Translated new copy on /languages/
- Added breadcrumbs
- Added h2 calling out ethereum.org as a language resource and the translation program

samajammin reviewed (approved):
Nice work! ❤️
